packages/webapp/src/components/Animals/AnimalBatchBasicInfo/index.tsx

```diff
@@ -25,11 +25,11 @@ import OtherDetails, { OtherDetailsProps } from '../../../components/Animals/Det
 
 interface AnimalBatchBasicInfoProps {
   isAnimal: boolean;
-  generalDetailProps: Omit<GeneralDetailsProps, 't' | 'animalOrBatch'>;
-  uniqueDetailsProps: Omit<UniqueDetailsProps, 't'>;
-  otherDetailsProps: Omit<OtherDetailsProps, 't' | 'animalOrBatch'>;
-  originProps: Omit<OriginProps, 't'>;
-  commonProps: CommonDetailsProps;
+  generalDetailProps: GeneralDetailsProps &
```

Comment:
I think `GeneralDetailsProps` and other types contains `CommonDetailsProps` and `animalOrBatch`, could you double check??

Reply:
Oh no, I even noticed the lack of type error without the union but ignored it 😳